convert_test.go

```diff
@@ -0,0 +1,234 @@
+package flatfs_test
+
+import (
+	"bytes"
+	"encoding/hex"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-ds-flatfs"
+	//"gx/ipfs/QmRWDav6mzWseLWeYfVd5fvUKiVe9xNH29YfMF438fG364/go-datastore/query"
+
+	rand "github.com/dustin/randbo"
```

Comment:
It isn't new dependency but PR introduces change that we know is not right. There is no cost in switching to `math/rand`.